public/modules/core/controllers/home.client.controller.js

```diff
@@ -144,29 +145,30 @@ angular.module('core').controller('HomeController', ['$scope', '$location', '$ro
                             _.each(trials[i].tumorTypes, function (tumorItem) {
                                 $scope.tumorTypes.push(tumorItem.tumorTypeId);
                             });
+                            if (alterationsFetched[i] !== undefined) {
+                                if (alterationsFetched[i].alterationsFetched.length > 0) {
```

Comment:
The same here, alterationsFetched[i]. alterationsFetched wont always be Array, right?